From 8a6852eafa0ee586e0cddef6e4108931e411f933 Mon Sep 17 00:00:00 2001
From: Claudiu Beznea <claudiu.beznea@microchip.com>
Date: Wed, 26 Jun 2019 15:51:16 +0300
Subject: [PATCH] wpa_cli_ifdef

Signed-off-by: Claudiu Beznea <claudiu.beznea@microchip.com>
---
 wpa_supplicant/wpa_cli.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/wpa_supplicant/wpa_cli.c b/wpa_supplicant/wpa_cli.c
index 05e3ebf2fb01..b6287ceca9ca 100644
--- a/wpa_supplicant/wpa_cli.c
+++ b/wpa_supplicant/wpa_cli.c
@@ -26,6 +26,9 @@
 #include <cutils/properties.h>
 #endif /* ANDROID */
 
+#ifndef CONFIG_P2P
+#define CONFIG_P2P
+#endif
 
 static const char *const wpa_cli_version =
 "wpa_cli v" VERSION_STR "\n"
-- 
2.21.0

